### PR TITLE
Added napalm_cli

### PR DESCRIPTION
**Pull request by dbarrosop**

### CODE REVIEW

**Comment by dbarrosop** [comment]
@yeled I have made a few changes to your code to make it more flexible and consistent with the rest of the modules.

@ktbyers if you are good with this I will merge and release 0.7.0

File: napalm_ansible/napalm_cli.py [hunk 1]
```diff
@@ -53,8 +57,10 @@
 else:
     napalm_found = True
 
+
 def main():
-    os_choices = ['eos', 'junos', 'ios', 'vyos', 'ros']
+    os_choices = ['eos', 'junos', 'iosxr', 'fortios', 'ibm',
```
Comment (ktbyers) [inline napalm_ansible/napalm_cli.py]: I though `ibm` driver was no-more?

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@ script:
     - ansible-playbook -i napalm_get_facts/hosts napalm_get_facts/get_facts_not_implemented.yaml -l multiple_facts.not_implemented -e "ignore_notimplemented=true"
     - ansible-playbook -i napalm_get_facts/hosts napalm_get_facts/get_facts_not_implemented.yaml -l multiple_facts.not_implemented -e "ignore_notimplemented=false"
     - ansible-playbook -i napalm_get_facts/hosts napalm_get_facts/get_facts_error.yaml -l multiple_facts.error
+    - ansible-playbook -i napalm_cli/hosts -l multiple_commands.ok napalm_cli/multiple_commands.yaml
+    - ansible-playbook -i napalm_cli/hosts -l wrong_commands.err napalm_cli/wrong_args.yaml
+    - ansible-playbook -i napalm_cli/hosts -l multiple_commands.ok napalm_cli/check_mode.yaml -C
```
Comment (ktbyers) [inline .travis.yml]: Can we add these to run_tests.sh (or have .travis.yml use run_tests.sh)? Basically, it is nice to have an easy way to manually run the tests.

Comment (ktbyers) [reply .travis.yml]: If the linter works now, shouldn't it be added to travis run?

Comment (dbarrosop) [reply .travis.yml]: > run_tests.sh

Yes, that's quite smart.

> linter

What do you mean?

File: napalm_ansible/napalm_cli.py (hunk 1 above)
Comment (dbarrosop) [reply napalm_ansible/napalm_cli.py]: Internsting, copy-pasted without paying attention. Will remove

**Comment by ktbyers** [comment]
+1

**Pull request merged**